dropshot/src/lib.rs

```diff
@@ -507,15 +507,35 @@
 //! this ought to work, but it currently doesn't due to serde_urlencoded#33,
 //! which is really serde#1183.
 //!
-//! ### DTrace probes
+//! Note that any parameters defined by `MyScanParams` are effectively encoded
+//! into the page token and need not be supplied invocations when `page_token`
+//! is specified. That is not the case for required parameters defined by
+//! `MyExtraQueryParams`--those must be supplied on each invocation.
+//!
+//! ### OpenAPI extension
+//!
+//! In generated OpenAPI documents, Dropshot adds the `x-dropshot-paginated`
+//! extension to paginated operations. The value is currently a structure
+//! with this format:
+//! ```json
```

Comment:
Does this render right without a newline here?

Reply:
<img width="949" alt="image" src="https://user-images.githubusercontent.com/677483/235572692-7ea067dd-b3e6-4719-8f93-1defd48d634a.png">

Yes it looks good to me.